### ISSUE TITLE
Free memory consumed by operation objects

### ISSUE THREAD

**Issue by jordan-heemskerk** [jordan-heemskerk/csc-462#39]
Make sure that we free any operation objects after they have... been applied? not entirely sure when

**Comment by eburdon** [comment]
Are we storing the 'Value' anywhere? I know the lab writeup said something about we're not supposed to store it in case its really large... but couldn't resolve why it was a problem. Maybe this is it.

**Comment by jordan-heemskerk** [comment]
Yeah, that's my problem. I guess we store it in the Op structs, but I think those should go out of scope fairly quickly and get GC'd
